presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -258,6 +254,32 @@ Property Name                                      Description
                                                    valid.
 ================================================== ============================================================ ============
 
+Hive Metastore Configuration Properties
+---------------------------------------
+
+The required Hive metastore can be configured with a number of properties.
+Specific properties can be used to further configure the :ref:`Thrift
+<thrift-metastore-properties>` or :ref:`Glue <glue-metastore-properties>`
+metastore.
+
+================================================== ============================================================ ============
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ============
+``hive.metastore``                                 The type of Hive metastore to use. Presto currently supports ``thrift``
+                                                   the default Hive Thrift metastore (``thrift``), and the AWS
+                                                   Glue Catalog (``glue``) as metadata sources.
+
+``hive.metastore-cache-ttl``                       Time to live Hive metadata cache.                            ``0s``
+
+``hive.metastore-refresh-interval``                How often to refresh the Hive metastore cache.
```

Comment:
> Asynchronously refresh cached metastore data after access
if it is older than this but is not yet expired, allowing
subsequent accesses to see fresh data.

Reply:
Done